cmd/kubeadm/app/util/apiclient/dryrunclient.go

```diff
@@ -185,7 +185,7 @@ func NewDryRunClientWithOpts(opts DryRunClientOptions) clientset.Interface {
 
 	// The chain of reactors will look like this:
 	// opts.PrependReactors | defaultReactorChain | opts.AppendReactors | client.Fake.ReactionChain (default reactors for the fake clientset)
-	fullReactorChain := append(opts.PrependReactors, defaultReactorChain...)
+	fullReactorChain := append(append([]core.Reactor(nil), opts.PrependReactors...), defaultReactorChain...)
```

Comment:
@sttts Thanks for spotting this!

Yet, it will be more readable if the two appends are on separate lines.